package.json

```diff
@@ -41,6 +42,7 @@
     "bn.js": "^5.2.1",
     "buffer": "^6.0.3",
     "chart.js": "^4.3.0",
+    "classnames": "^2.3.2",
```

Comment:
Styled components can be used in the dashboard. So this dependency is not needed.